### ISSUE TITLE
Resource class keys are incorrect

### ISSUE THREAD

**Issue by levlaz** [circleci/circleci-docs#3747]
# Description
A customer reported that the keys in the documentation do not work for extra-large resources. 

For example, the documentation states that 2XL+ is a valid key, but when you try to use CircleCI will show the following error: 

```
Resource class 2XL+ is not a valid resource class. Valid classes: small, medium, medium+, large, xlarge, 2xlarge, 2xlarge+. Default class medium will be used.
```

# Location
https://circleci.com/docs/2.0/configuration-reference/#resource_class

# Steps to Reproduce
Set `resource_class: 2XL+` in a build.

**Comment by rosieyohannan** [comment]
Hi  @levlaz, thanks for pointing this out! We're working on sorting out the issue, but for now I've hidden these resource classes from the config reference. I will update here when the issue is fixed. Thanks! Rosie

**Comment by mvxt** [comment]
This would be addressed in this PR: https://github.com/circleci/circleci-docs/pull/3746

**Comment by mvxt** [comment]
#3746

**Comment by mvxt** [comment]
Fixed in PR: #3746

Deployed to prod: https://circleci.com/docs/2.0/configuration-reference/#resource_class
